docs/mailbox.md

```diff
@@ -0,0 +1,10 @@
+## Message to mailbox
+### The TruckAgent informs the mailbox about the delivered order
+#### String: "OrderID"
+<br>
+
+
+## Message from the mailbox
+### The mail box informs the concerned customer and other agents who are interested in knowing the delivery status of the order
+#### String: "OrderID"
```

Comment:
What about the time it was delivered?

Reply:
The time and date of delivery is also provided in the updated message description